README.md

```diff
@@ -308,9 +308,7 @@ But this comes with some nasty caveats and limitations:
 # Design rationale behind filter-repo
 
 None of the existing repository filtering tools did what I wanted;
-they all came up short for my needs.  No tool provided any of the
-first eight traits below I wanted, and all failed to provide at least
-one of the last four traits as well:
+they all came up short for my needs. No tool provided any of the first eight traits below I wanted, and no tool provided more than two of the last four traits either:
```

Comment:
Could you wrap this long line like the rest of the document?  After that, I'll be happy to apply.

Reply:
Ah yeah sorry about that -done now